### PR TITLE
Samsung backend update for sub v3.1

### PR DESCRIPTION
**Pull request by AhmedTElthakeb**

### CODE REVIEW

**Comment by AhmedTElthakeb** [comment]
will fix the formatting issue shortly and update

File: mobile_back_samsung/samsung/lib/public/include/mbe_config_1200.pbtxt
Comment (anhappdev) [inline mobile_back_samsung/samsung/lib/public/include/mbe_config_1200.pbtxt:1]: You may want to add this annotation to the beginning of every `.pbtxt` file. It helps the IDE recognize the schema and enable features like auto-completion, validation.

```
# proto-file: flutter/cpp/proto/backend_setting.proto
# proto-message: BackendSetting
```

**Comment by anhappdev** [comment]
@AhmedTElthakeb I uploaded the new Samsung libs to our CI storage in https://github.com/mlcommons/mobile_app_open/pull/763. Please review and merge it, then after the CI build is completed, please also download and test the unified APK generated by the CI. Thank you.

**Review by anhappdev — APPROVED** [review]

**Review by mjkim010 — APPROVED** [review]
Thanks a lot, Ahmed.

**Pull request merged**